src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -203,7 +203,7 @@ def unshape(x):
         q = q / math.sqrt(dim_per_head)  # (bs, n_heads, q_length, dim_per_head)
         scores = torch.matmul(q, k.transpose(2, 3))  # (bs, n_heads, q_length, k_length)
         mask = (mask == 0).view(mask_reshp).expand_as(scores)  # (bs, n_heads, q_length, k_length)
-        scores.masked_fill_(mask, -float("inf"))  # (bs, n_heads, q_length, k_length)
+        scores = scores.masked_fill(mask, -float("inf"))  # (bs, n_heads, q_length, k_length)
```

Comment:
@sgugger @LysandreJik it's a tiny modification but wanted to make sure it's okay on your end?

The reason for this is because tracing with torch.fx seems to ignore this op because it is inplace, making predictions wrong due to not masking the scores at all.

Reply:
I think this is fine, but it looks to be a bug in `torch.fx` ?

Reply:
Let me investigate on that and come back to you